### PR TITLE
The Newyork Times Article

### PR DESCRIPTION
**Pull request by imran5644**
## the Newyork times clone
in this project we used :
1 Navigation bar
2. floats
3. position
4. grid
5. flexbox
6. footer
7 . aside section
8. images and video

### CODE REVIEW

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

Team, good job so far with this project! 👍 
Your linters are green 🟢 
Your code is clean ! 👏 
You use semantic HTML tags properly 💯 

However, there are some things to improve before approving the project! 
Check my comments below 👇 , apply the required changes and resubmit your project for a new code review 📑 

Good luck with it, keep the good work up ! 😃 

If you have any question or doubt, reach me out on Slack.

## CHANGES

- Your website has an horizontal scrollbar, please check your containers to check which one is causing that and modify it
- Navbar
   - 'Space and cosmos' and the new york times logo must be a link. Kindly fix it
   - Respect the height of the original
   - Please, modify the positioning of the subscribe and login buttons to make them equal to the original, now there is too much space from the top and right side of the screen.
- The text under the main image should be align with the image.
- Main content must be centered, now it is not completely centered, please fix it
- Please modify font size of the main content match the original
- Kindly, modify the space between paragraphs by adding a bit of margin to them.
- In the most popular articles section, please modify the font size and the padding, make it similar to the original
- Kindly fix the aside section, should be equal to the original.
- All the social media icons should be links or at least cursor pointer. Also add them the background and the hover effect.
- In your readme file, acknowledgments section, do not show your links, add text as a link, just like with your live demo link

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle:

### Hi team :wave: 

**So far, this is looking good. However, some of the issues raised by the previous TSE is yet to be fixed. Let's break things down again:**

**[REQUIRED CHANGES]**

- [x] Your page is still having a horizontal scroll bar. I'm sure this is caused by one of your paddings or margins from one of your containers. Inspect your page and I believe you should be able to figure out which of the containers is causing the horizontal scroll bar. Please Identify and fix this issue.

- [x] Observe the first title of your page (check the image below). If at all the title will have to break to the next line, it should align vertically to the left with the first line. 
- From my observation, I can see that you have a separate container for the first line (Space Ripples Reveal Big Bang’s) and another one for the second line (Smoking Gun). This should not be so. The second line is a continuation of the first line. Check the Images below for better understanding:

**Original page** 

o

**Your clone**

c

Kindly fix this.

- [x] Compare your `Most Popular` section with the original. Please try to make it look like the original.
- The original version of the `Most Popular` section is having some white spaces to the right. Kindly find a way to add it.
- The link items in the `Most Popular` section are not bolded in the original. Please adjust this.

**Original `Most Popular` section:**

o


**Your cloner version of `Most Popular` section:**

y

**Kindly make all the changes required and request another review when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**

Happy Coding :man_technologist:

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:
## :wave: :wave: Hello team!! Great effort on the project!!

There are still unmet requirements. We should focus on improving the suggested changes.

## Requested changes

### [Mandatory] :red_circle: 

- [x] Image not showing properly. It shows disproportionally.
image

- [x] Not centered
image

- [x] Overflowing section.
image

- [x] Horizontal scroll. This could be related to the previous issue.

- [x] NYT logo is not a link
image

- [x] Images should be links.
image

- [x] Use ```a``` html link tag instead of ```cursor: pointer```. Update elements to links.

### [OPTIONAL]  :large_blue_circle: 

- [ ] Keep CSS DRY. see inline comments

### [Comments] :green_circle: 

- The browser **Inspector Mode** is a great tool to understand what's the website doing under the hood. Also, a way to see if the project will look as intended for bigger or smaller screens. 
The Project has more issues than commented, but they are out of the scope of the project objective, as it doesn't ask us to review for responsiveness. **You may or may not work in this issues, is totally up to you**, but if this persist will cause issues in future projects. To give you an idea:
image
image
image
image
image


## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: stylesheets/style.css
Comment (jubaan) [inline stylesheets/style.css]: DRY stands for "Don't Repeat Yourself". Keeping the project code dry helps readability and makes it easier to find bugs. Example of how to keep the code DRY.
Improved version of the code above:
```
.name, 
.aside {
    display: none;
  }
```

**Review by jubaan — CHANGES_REQUESTED** [review]

**Review by teekaytech — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## Hi @imran5644  & @marcelomaidden  :wave: 

### You guys have really done a lot to improve his project. Almost all the changes required by the previous reviewers have been fixed and for that, I commend you :+1:

**However, we still need to make one or two changes before this project can be approved. Kindly check my points below and fix them as necessary.:**

**[REQUIRED CHANGES]**

- [ ] Your live demo link is outdated. The current one does not reflect the recent changes you've made. Kindly update your live demo link.

- [ ] There's a line under the `Editors' Pick` section. This is not the case in the original version. Kindly remove the line.

Here's your clone

o


...and here's the original

y


**Kindly make all the changes required and request another review when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**

Happy Coding :man_technologist:

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations team, this project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:

**Comment(s)**
- In your next project, take some time to understand the project requirements well and pay more attention to details. It will help you a great deal as you move on in your journey to become a world-class developer.

Again, congratulations, you can merge to master.
Good luck with your next project.

**I'm @taofeekolalere on slack.
Happy Coding :man_technologist:**

**Pull request merged**
